Subject: [PATCH] Patch terra-rebels wallet controller
---
Index: node_modules/@terra-rebels/wallet-controller/_commonjs/operators/getExtensions.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/node_modules/@terra-rebels/wallet-controller/_commonjs/operators/getExtensions.js b/node_modules/@terra-rebels/wallet-controller/_commonjs/operators/getExtensions.js
--- a/node_modules/@terra-rebels/wallet-controller/_commonjs/operators/getExtensions.js	(revision 80e4c9a2415ceffd590997fece5a7ca2ae6dccfa)
+++ b/node_modules/@terra-rebels/wallet-controller/_commonjs/operators/getExtensions.js	(date 1697145916860)
@@ -40,7 +40,7 @@
     if (!currentBrowser) {
         return (0, rxjs_1.of)([]);
     }
-    return (0, fetch_1.fromFetch)('https://assets.terrarebels.net/extensions.json').pipe((0, rxjs_1.switchMap)((res) => {
+    return (0, fetch_1.fromFetch)('https://assets.hexxagon.io/extensions.json').pipe((0, rxjs_1.switchMap)((res) => {
         if (res.ok) {
             return res.json();
         }
Index: node_modules/@terra-rebels/wallet-controller/operators/getExtensions.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/node_modules/@terra-rebels/wallet-controller/operators/getExtensions.js b/node_modules/@terra-rebels/wallet-controller/operators/getExtensions.js
--- a/node_modules/@terra-rebels/wallet-controller/operators/getExtensions.js	(revision 80e4c9a2415ceffd590997fece5a7ca2ae6dccfa)
+++ b/node_modules/@terra-rebels/wallet-controller/operators/getExtensions.js	(date 1697145916863)
@@ -7,7 +7,7 @@
         {
             name: 'Rebel Station',
             identifier: 'rebel-station',
-            icon: 'https://assets.terrarebels.net/icon/station-extension/icon.png',
+            icon: 'https://assets.hexxagon.io/icon/station-extension/icon.png',
             urls: [
                 {
                     browser: 'chrome',
@@ -37,7 +37,7 @@
     if (!currentBrowser) {
         return of([]);
     }
-    return fromFetch('https://assets.terrarebels.net/extensions.json').pipe(switchMap((res) => {
+    return fromFetch('https://assets.hexxagon.io/extensions.json').pipe(switchMap((res) => {
         if (res.ok) {
             return res.json();
         }
